src/client/java/teammates/client/scripts/sql/DataMigrationForStudentSql.java

```diff
@@ -0,0 +1,92 @@
+package teammates.client.scripts.sql;
+
+import com.googlecode.objectify.cmd.Query;
+
+// import jakarta.persistence.criteria.CriteriaBuilder;
```

Comment:
Any reason why you leave them as comments?

Reply:
For convenience, to be uncommented along with private `getTeam` function below after the migration script for `Team` entities is done.